src/celttf/truetypefont.cpp

```diff
@@ -113,15 +117,14 @@ struct TextureFontPrivate
     bool                    m_shaderInUse{ false };
     std::vector<FontVertex> m_fontVertices;
 
-    static GLuint m_vbo;
-    static GLuint m_vio;
+    gl::VertexObject m_vao;
+    gl::Buffer       m_vbo{ gl::Buffer::TargetHint::Array };
```

Comment:
Since the `gl::Buffer` is non-copyable, the copy constructor and assignment operator on `TextureFontPrivate` should be switched to `delete` rather than `default` (causes a warning on clang)